JavaFXUserInterface/src/main/resources/css/revamp/assetType.css

```diff
@@ -40,4 +40,157 @@
     -fx-background-color: #FBFBFC;
     -fx-border-width: 1px;
     -fx-border-color: #E8E9EA transparent #E8E9EA transparent;
+}
+
+.tab
+{
+    -fx-background-color: transparent;
+    -fx-font-family: 'Segoe UI';
```

Comment:
is there a way to define a global font family for the project, and remove all individual references?

Reply:
https://stackoverflow.com/questions/18408884/set-font-globally-in-javafx/18409438#18409438

Reply:
This will be fixed in the CSS clean up issue